### ISSUE TITLE
Add Break Glass account

### ISSUE THREAD

**Issue by WilliamDormer** [PullDating/Org#25]
In case I fall off a cliff and cannot maintain the repo, then the core team should be able to recover the project.

**Comment by flo-kei** [comment]
@WilliamDormer any updates on this? (now that the project's alive again, this might be an issue worth looking after, at least that's my take :)

**Comment by WilliamDormer** [comment]
Yes this is an important issue. At the moment there's not much to worry about, but once we start having some live deployments, or some external collaboration tools (e.g. Penpot) we'll have to figure out how to do that.
